testcontainers/core/policies/default.py

```diff
@@ -0,0 +1,13 @@
+from docker.models.images import Image
+
+from testcontainers.core.policies.base import ImagePullPolicy
+
+
+class DefaultPullPolicy(ImagePullPolicy):
+    """
+    The default ImagePullPolicy, which pulls the image from a remote repository
+    only if it does not exist locally
+    """
+
+    def should_pull_cached(self, image: Image) -> bool:
```

Comment:
Because `ImagePullPolicy` is abstract.

Reply:
My hunch is that we can remove the notion of an abstract `ImagePullPolicy` because python uses duck-typing (as opposed to more interface-heavy Java, for example). Some discussion [here](https://stackoverflow.com/a/372121).